packages/client/ui/react-ui/src/hooks/useCrossmint.tsx

```diff
@@ -45,10 +46,10 @@ export function CrossmintProvider({
     return <CrossmintContext.Provider value={value}>{children}</CrossmintContext.Provider>;
 }
 
-export function useCrossmint() {
+export function useCrossmint(missingContextMessage?: string) {
```

Comment:
So that when this is used within a specific package, we can throw a package specific error. Example: `'CrossmintMintingProvider' must be used within the base 'CrossmintProvider', see xxx docs`